examples/objectServerExample/build.gradle

```diff
@@ -34,18 +14,20 @@ android {
     }
 
     buildTypes {
-        // This will automatically try to detect the IP address of the machine
-        // building the example. It is assumed that this machine is also running
-        // the Object Server. If not, replace 'host' with the IP of the machine
-        // hosting the server. In some cases the wrong IP address will also
-        // be detected. In that case also insert the IP address manually.
-        def host = getIP()
+        // Go to https://cloud.realm.io and copy the URL to your instance. Insert it below.
+        // It will look something like "https://test.us1.cloud.realm.io"
```

Comment:
Add a section about self-hosted:

If you're running a self-hosted version, use the hostname/IP address of the Realm Object Server (example: `http://192.168.1.64:9080`